lib/matplotlib/axis.py

```diff
@@ -1823,9 +1823,17 @@ def set_ticks(self, ticks, labels=None, *, minor=False, **kwargs):
         Parameters
         ----------
         ticks : list of floats
-            List of tick locations.
+            List of tick locations.  The axis `.Locator` is replaced by a
+            `.FixedLocator`.
```

Comment:
```suggestion
            `~.ticker.FixedLocator`.
```
Will fix the doc build (cause: There's also `mpl_toolkits.axisartist.grid_finder.FixedLocator`, so Sphinx doesn't know which one to link).